src/cmd/cli/command/commands.go

```diff
@@ -129,10 +129,10 @@ func Execute(ctx context.Context) error {
 
 func SetupCommands(version string) {
 	RootCmd.Version = version
-	RootCmd.PersistentFlags().Var(&colorMode, "color", `colorize output; "auto", "always" or "never"`)
+	RootCmd.PersistentFlags().Var(&colorMode, "color", `colorize output; one of [always never]`)
```

Comment:
I find space-delimiting the options to be a little confusing. I think using pipes (`|`) might be more typical. What do you think of
```suggestion
	RootCmd.PersistentFlags().Var(&colorMode, "color", `colorize output; one of [always|never]`)
```

Reply:
There's quite a few places in the CLI where we print `one of [foo bar]` so I'd want to change all of them then. I'd rather do this in a separate PR.